src/System.Text.Json/tests/Serialization/EnumTests.cs

```diff
@@ -39,6 +39,83 @@ public static class EnumTests
                 @"}";
 
         private const string UlongMaxPlus1 = "18446744073709551616"; // ulong.MaxValue + 1;
+        private const string MinusUlongMaxMinus1 = "-18446744073709551616"; // -ulong.MaxValue - 1
+
+        private const string LongMaxPlus1 = "9223372036854775808"; // long.MaxValue + 1;
+        private const string MinuslongMaxMinus1 = "-9223372036854775809"; // long.MinValue - 1
```

Comment:
nit: Maybe change these to the (U)Int32, (U)Int64 names rather than keyword names like long (makes PascalCasing cleaner).